enterprise/server/test/integration/workflow/workflow_test.go

```diff
@@ -31,8 +32,13 @@ import (
 	wfpb "github.com/buildbuddy-io/buildbuddy/proto/workflow"
 )
 
-var (
-	simpleRepoContents = map[string]string{
+// repoContents simulates a test repo with the config files required to run a workflow
+func repoContents() map[string]string {
```

Comment:
`repoContents` is a bit vague. What do you think about calling this `simpleRepo` and renaming repoContentsSlowScript to `slowScriptRepo` or `repoWithslowScript`?

Reply:
Done